addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -6733,6 +6773,23 @@ registry.ImageTools = ImageHandlerOption.extend({
      */
     async _writeShape(svgText) {
         const img = this._getImg();
+
+        // Apply the right animation speed if there is an element animate.
+        
+        if (this._isAnimatedShape() && !img.dataset.hoverEffect) {
+            if (!img.dataset.animationRatio) {
+                img.dataset.animationRatio = this.currentAnimationRatio || "5";
+            }
+            svgText = svgText.replace(/((\d+(\.\d+)?)|(\.\d+))s([\"\s])/gm, function(matched) {
+                const ratio = (10 ** ((img.dataset.animationRatio - 5) / 5)).toFixed(1);
```

Comment:
I'm agree with you and I've change my mind with the logarithm scale. That's too weird. I opted to design a slider spanning from -1000% (10 times slower) to +1000% (10 times faster). The inclusion of the sign is crucial in this context. This approach enables me to avoid utilizing a logarithmic scale, which would unnecessarily complicate the implementation process.